src/main.rs

```diff
@@ -118,14 +118,28 @@ fn main() -> Result<()> {
     let mut report_data = [0u8; 64];
     report_data[0..32].copy_from_slice(report_binding.as_ref());
 
```

Comment:
Hard to follow the logic with all the compile flags

Reply:
I am simply disabling the attestation/collateral creation if we are not in an SGX environment. I had no choice than specifying the compile flags like this.

Reply:
But I guess it can be refactored